stories/modal-flow.js

```diff
@@ -0,0 +1,111 @@
+import React, { Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+import { withReadme } from 'storybook-readme';
+import { State, Store } from '@sambego/storybook-state';
+import { Modal, ModalFlow, ModalStep } from '../src';
+import readme from '../src/components/modal-flow/README.md';
+
+Modal.setAppElement('#root');
+
+const steps = [
+    {
+        id: 'first',
+        content: 'This is the first step.',
+        goToId: 'second',
+    },
+    {
+        id: 'second',
+        content: 'This is the second step.',
+        goToId: 'third',
+    },
+    {
+        id: 'third',
+        content: 'This is the third step.',
+        goToId: 'fourth',
+    },
+    {
+        id: 'fourth',
+        content: 'This is the fourth step.',
+        goToId: 'last',
+    },
+    {
+        id: 'last',
+        content: 'This is the last step.',
+        goToId: 'third',
+    },
+];
+
+storiesOf('Modal Flow', module)
+.addDecorator(withReadme(readme))
+.add('Simple', () => {
+    const store = new Store({
+        stepId: 'first',
+    });
+
+    return (
+        <Fragment>
```

Comment:
This fragment should not be required as you have a single component State as its child